selfhosted/queues.py

```diff
@@ -0,0 +1,17 @@
+import pathlib
+
+import click
+import yaml
```

Comment:
If you save the queues in JSON (the file is extremely simple) and use `argparse` instead of `click`, you can avoid all external dependencies, and just work with the standard library.

Reply:
Actually, if we use JSON maybe we can get read of `queues.py` completely and do it directly in the shell, using `jq` or some [actions trick](https://stackoverflow.com/questions/61919141/read-json-file-in-github-actions). However, `jq` is not available in the cluster, so it is probably not an option, and I am not sure if the actions allow indexing the loaded JSON with a variable (in our case the platform `name`). Also, these solutions are generally harder to implement and read than Python.

If we keep Python, I agree regarding the dependencies.